extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -1175,3 +1175,38 @@ export const BlockaidMaliciousTxWarning = ({
     </WarningMessage>
   );
 };
+
+export const BlockaidMaliciousTxInternalWarning = ({
+  description,
+}: {
+  description: string;
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="ScamAssetWarning__box" data-testid="ScamAssetWarning__box">
+      <div className="Icon">
+        <img
+          className="ScamAssetWarning__box__icon"
+          src={IconWarningBlockaid}
+          alt="icon warning blockaid"
+        />
+      </div>
+      <div>
+        <div className="ScamAssetWarning__description">
+          {t(
+            "This transaction was flagged by Blockaid for the following reasons:",
+          )}
+          <div>{description}</div>
```

Comment:
will take the description straight from Blockaid, so it can be something like
`Gaining account GA223OFHVKVAH2NBXP4AURJRVJTSOVHGBMKJNL6GRJWNN4SARVGSITYG is classified as malicious_link`